.github/ISSUE_TEMPLATE/content.md

```diff
@@ -8,13 +8,16 @@ assignees: ''
 
 Link or add dependencies to any applicable issues.
 
-Include any links or relevant information from Slack conversations that related to this issue.
+Include any links or relevant information from Slack conversations that related
+to this issue.
```

Comment:
This is strange to have a line break here. is prettier modifying this markdown?

Reply:
Yes, I can tell it not to, it's setting the line length